core/opendaq/reader/tests/test_multi_reader.cpp

```diff
@@ -2016,3 +2031,30 @@ TEST_F(MultiReaderTest, MultiReaderExcetionOnConstructor)
         sig0.createAndSendPacket(i);
     }
 }
+
```

Comment:
I would love to see a test with gaps in data. It should produce gap packets in multi reader.